packages/expo-module-template-local/android/src/main/java/{%= project.package %}/{%- project.viewName %}.kt

```diff
@@ -1,7 +1,28 @@
 package <%- project.package %>
 
 import android.content.Context
+import android.webkit.WebView
+import android.webkit.WebViewClient
 import expo.modules.kotlin.AppContext
+import expo.modules.kotlin.viewevent.EventDispatcher
 import expo.modules.kotlin.views.ExpoView
 
-class <%- project.viewName %>(context: Context, appContext: AppContext) : ExpoView(context, appContext)
+class <%- project.viewName %>(context: Context, appContext: AppContext) : ExpoView(context, appContext) {
+    // Creates and initializes an event dispatcher for the `onLoad` event.
+    // The name of the event is inferred from the value and needs to match the event name defined in the module.
+    private val onLoad by EventDispatcher()
+
+    // Defines a WebView that will be used as the root subview.
+    internal val webView = WebView(context).also {
```

Comment:
```suggestion
    internal val webView = WebView(context).apply {
```
Apply would be a better choice here.

Reply:
https://docs.expo.dev/modules/native-view-tutorial/ we should also update the tutorial if so